autoextract/aio/client.py

```diff
@@ -175,10 +202,17 @@ async def request_raw(query: Query,
     ``agg_stats`` argument allows to keep track of various stats; pass an
     ``AggStats`` instance, and it'll be updated.
 
+    Additional ``headers`` for the requests can be provided.
+
+    Default retry policy can be customized by providing a
+    ``retry_wrapper`` that can be built with a customized
+    :class:`autoextract.retry.RetryFactory`
```

Comment:
Why is it called retry_wrapper? As I understand, this function receives arbitrary tenacity.Retrying object; autoextract.retry.RetryFactory can be used as a starting point/helper to build customized Retrying object for AutoExtract.

Reply:
What do you think about calling it `retry_policy`, `retry_strategy` or something like that? Document that it should be a tenacity.Retrying instance, and that :class:`autoextract.retry.RetryFactory` helper exist.

Reply:
Yeah, I was in doubt about how to name it. The reason to call it `retry_wrapper` is that it is actually just a function wrapper what is required. It doesn't even require to be a tenacity wrapper. See the tenacity code here: https://github.com/jd/tenacity/blob/master/tenacity/__init__.py#L109 

> Wrap a function with a new `Retrying` object.

A wrapper like the following would also be acceptable:
```py
def simple_retry(count):
  def fn(f):
    for _ in range(count):
      try:
        return f()
      except Exception as e:
        pass
    raise e
```
So you could pass `simple_rety` as the `retry_wrapper`.

If you have a suggestion about a better name it will be welcome!

Reply:
Ohh, sorry, now I see your suggestions: `retry_policy`, `retry_strategy`. I'll think about it.

Reply:
Ok, so finally I decided to clarify the documentation:

https://github.com/scrapinghub/scrapinghub-autoextract/pull/24/commits/ebcb1318687d28a612108fae0831b3ba3e00b5f2